### ISSUE TITLE
Wrong Font in List of Figures and list of Tables

### ISSUE THREAD

**Issue by TinyRbrt** [derric/cleanthesis#86]
Font for headers of List of Tables and list of Figures is not the same font, as set throughout the document. I added the following to the corresponding part in cleanthesis.sty
```
\RequirePackage{tocloft}
%
% > adjusts the indention for figure and table entries (lof and lot)
%   -- \cftsetindents{entry}{indent}{numwidth} sets the entry's indent to
%      the length indent and its numwidth to the length numwidth
\cftsetindents{figure}{0em}{3em}
\cftsetindents{table}{0em}{3em}
%%%%%%These are the new commands to fix it%%%%%
\renewcommand\cftloftitlefont{\usekomafont{chapter}}%change list of figures font to chapter-font
\renewcommand\cftlottitlefont{\usekomafont{chapter}}%change list of tables font to chapter-font
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%
```

**Comment by ecasglez** [comment]
I also had to add the following for the "Contens" font.

```\renewcommand\cfttoctitlefont{\usekomafont{chapter}}```

However, in the main website of cleanthesis there is an example which shows the right font: http://cleanthesis.der-ric.de/img/ct_toc.jpg
I do not know why in my case it used a different font for the "Contents" word.

**Comment by ecasglez** [comment]
Now I see this is fixed in the develop branch, in commits e62216bb5962460359f4c9e45f13869b27d6315f and a331a3d71b6e554f275058d0ab793988cc6b3c78
